packages/api/tests/endpoints/test-granules.js

```diff
@@ -5,6 +5,7 @@ const request = require('supertest');
 const path = require('path');
 const sinon = require('sinon');
 const test = require('ava');
+const omit = require('lodash/omit');
```

Comment:
The updates in this test file refactor the original move tests and add cases for failures/etc, reusing code/boilerplate where possible.  They are intended to be comprehensive within the scope of the unit test environment to the api endpoint/lib code.